Task1/Program.cs

```diff
@@ -10,7 +10,130 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello world!");
+            /* Console.WriteLine("Add Radius");
+
+             string radiusString = Console.ReadLine();
+             int.TryParse(radiusString, out int radius);
+
+             Console.WriteLine($"The square of the round {Program.PerformCalculation(15)}");*/
+
+            //HT1_1();
+            //HT1_2();
+            //HT1_3();
+            //HT1_4();
+            //HT1_5();
+            //HT1_6();
+            //HT1_7();
+            //HT1_8();
+            //HT1_9a();
+            HT1_9b();
+
+        }
+
+        public static decimal PerformCalculation(int r)
+        {
+            const decimal Pi = 3.14m;
+
+            decimal square = Pi * r * r;
+
+            return square;
+        }
+
+        public static void HT1_1()
+        {
+            int x1 = 5;
+            Console.WriteLine(x1);
+        }
+
+        public static void HT1_2()
+        {
+            Console.WriteLine("Enter your name: ");
+            string str1 = Console.ReadLine();
+            string str2 = "Hi, " + str1;
```

Comment:
Since you are known with string interpolation ($"{}"), lets use that approach everywhere.